packages/core/src/Stores/client-store.js

```diff
@@ -1584,6 +1584,29 @@ export default class ClientStore extends BaseStore {
         this.registerReactions();
         this.setIsLoggingIn(false);
         this.setInitialized(true);
+
+        const unusedParams = [
+            'type',
+            'acp',
+            'label',
+            'server',
+            'interface',
+            'cid',
+            'age',
+            'utm_source',
+            'first_name',
+            'second_name',
+            'email',
+            'phone',
+            '_filteredParams',
+        ];
+
+        const all_search_params = new URLSearchParams(search);
+        const filtered_queries = [...all_search_params].filter(kvp => !unusedParams.includes(kvp[0]));
```

Comment:
What is `kvp`? can we use a more descriptive word?

Reply:
kvp is a key value pair, but I'll change it to [...all_search_params].filter(([key]) => !unusedParams.includes(key));